components/automate-deployment/pkg/server/helpers.go

```diff
@@ -0,0 +1,27 @@
+package server
+
+import (
+	"bytes"
+	"fmt"
+)
+
+func LogRotateConf(path string, params ...string) string {
+	if len(params) < 1 {
+		return ""
+	}
+	var buffer bytes.Buffer
+	buffer.WriteString(path + " {" + "\n")
+	for _, param := range params {
+		buffer.WriteString("\t" + param + "\n")
+	}
+	buffer.WriteString("}" + "\n")
+	return buffer.String()
+}
+
+func getLogFileName(path string) string {
+	return fmt.Sprintf("%sautomate.log", path)
```

Comment:
there can be a possibility that the user doesn't provide the `/` at the end. So please make sure to support that as well.